numba/pythonapi.py

```diff
@@ -1581,3 +1582,65 @@ def string_from_constant_string(self, string):
         cstr = self.context.insert_const_string(self.module, string)
         sz = self.context.get_constant(types.intp, len(string))
         return self.string_from_string_and_size(cstr, sz)
+
+    def call_jit_code(self, func, sig, args):
```

Comment:
Am wondering if there's use cases where a function may already be jit compiled? For example an argument checking routine that is reused in multiple places.

Reply:
The trouble is that the user need to write a lot of code to resolve the true function value. It seems to be easier just to write it in jit-code.

Reply:
Fair enough!